src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -56,7 +56,7 @@ public void execute_invalidCommandFormat_throwsParseException() {
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
         String deleteCommand = "delete 9";
-        assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        assertCommandException(deleteCommand, MESSAGE_INVALID_TRANSACTION_DISPLAYED_INDEX);
     }
```

Comment:
Not sure if this was a refactor error. I don't think that the index should be an invalid transaction index here? Wrong command.

Reply:
Thanks for noticing, fixed.